Description: drop ruby-static feature
 libruby does not provide libruby-static in debian.
Author: Ananthu C V <weepingclown@disroot.org>
Forwarded: not-needed
Last-Update: 2024-05-01
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -47,7 +47,6 @@
 link-ruby = []
 no-link-ruby = []
 ruby-macros = ["stable-api"]
-ruby-static = []
 stable-api = []
 stable-api-compiled-fallback = ["stable-api"]
 stable-api-compiled-force = []
